src/command_executor.cc

```diff
@@ -926,6 +927,31 @@ sql_callback(exec_context& ec, sqlite3_stmt* stmt)
             dls.push_header(colname, type, graphable);
             if (graphable) {
                 auto attrs = vc.attrs_for_ident(colname);
+                dls.dls_headers[lpc].hm_colorstr = colname;
+                for (
+                    std::string suf = " ";
+                    std::any_of(
+                        dls.fg_colors_vec.cend() - ( std::min((int)dls.fg_colors_vec.size(), (int) (view_colors::HI_COLOR_COUNT - 1))),
+                        dls.fg_colors_vec.cend(),
+                        [attrs](auto el) {return el == attrs.ta_fg_color;}
```

Comment:
Note that the capture `[attrs]` is going to copy the object, might be better to capture by reference by prefixing it with `&` (so, `[&attrs]`).

Reply:
I agree, thanks for pointing that out